nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -240,8 +234,17 @@ define([
         if (options.initialize) {
             initialize(options.specs);
         }
+
         const model = Props.make({
-                data: Utils.getMeta(cell, 'bulkImportCell'),
+                // TODO: Remove and replace with the commented-out line below
+                // once the backend is hooked up
+                data: Object.assign(
+                    {},
+                    Utils.getMeta(cell, 'bulkImportCell'),
+                    // execution data
+                    { exec: TestAppObj.exec }
```

Comment:
we have all the rest of the data, so just supply the `exec` (job) info from the TestAppObject